src/_locales/en/messages.json

```diff
@@ -842,6 +846,10 @@
     "message": "Opens a new empty private window.",
     "description": "Opens a new empty private window."
   },
+  "commandDescriptionFullscreen": {
+    "message": "Sets the window to its full size.",
+    "description": "Sets the window to its full size."
```

Comment:
Please change the description to: "Activates full screen mode for the current window." The main reason for this change is that "full size" might be misunderstood as "maximize the window"